### ISSUE TITLE
https://wiki.developer.mozilla.org/zh-CN/docs/Web/CSS/CSS_Flexible_Box_Layout/Controlling_Ratios_of_Flex_Items_Along_the_Main_Ax

### ISSUE THREAD

**Issue by hespoon** [mdn/sprints#2714]
# Request type

- [ ] Please close this issue, I accidentally submitted it without adding any details
- [ ] New documentation
- [x] Correction or update

# Details

文章中的 “flex-basis 属性” 部分，第一段，第三行的第二个 “flex“ 单词拼写错误。原文如下：”比如说你已经给你的felx子元素设置了200px 的宽“

**Comment by SphinxKnight** [comment]
Hello, could you rephrase what needs to be fixed in English? 
In any case, do not hesitate to edit the page by yourself to fix this (MDN is a wiki anyone can edit :))
